recipes/emperor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "3.8.2" %}
```

Comment:
Sorry, this is a vestige of the tutorial/template provided by the sample
recipe. Should be fixed now.

On (Jul-25-16|21:41), jakirkham wrote:
> > @@ -0,0 +1,48 @@
> > +{% set version = "3.8.2" %}
> 
> This version seems to not be available via PyPI. 😕
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/conda-forge/staged-recipes/pull/1070/files/ce02e997e03c63a0e2e577e53a0deb91946edaf3#r72187214